Python/Product/Cookiecutter/Model/CookiecutterClient.cs

```diff
@@ -131,6 +141,102 @@ class CookiecutterClient : ICookiecutterClient {
             MoveToDesiredFolder(outputFolderPath, tempFolder);
         }
 
+        private static void LoadVisualStudioSpecificContext(List<ContextItem> items, JProperty vsExtrasProp) {
+            // This section reads additional metadata for Visual Studio.
+            // All fields are optional, but if they are specified, they are validated.
+            //
+            // Example:
+            // "_visual_studio" : {
+            //   "var1" : {
+            //     "description" : "description for var 1",
+            //     "type" : "yesno"
```

Comment:
I prefer the term "selector" here, so that it won't be confused with the Python type of the result.